src/server/cea_608_encoder/byte_pair_generator.py

```diff
@@ -67,14 +68,16 @@ def consume_scenes(scene_list: list) -> list:
     :return: TODO
     """
     scene_data = []
-    
+
     for scene in scene_list:
         current_scene_data = {}
         current_scene_data['start'] = 0
         current_scene_data['data'] = []
 
         if 'scene_id' not in scene:
             raise ValueError('Every scene must have a scene ID.')
+        else:
+            validate_scene_ids(scene_list) 
```

Comment:
This and the caption id validation call are happening within for-loops so for each scene were gonna validate every single scene id. If we have 100 scenes, we will perform the same validation 100 times ^^

Reply:
Where do you recommend I put it?

Reply:
Probably at the end of the for-loop, outside of it.